pkg/skaffold/server/server.go

```diff
@@ -231,21 +259,6 @@ func newHTTPServer(preferredPort, proxyPort int, usedPorts *util.PortSet) (func(
 	}, nil
 }
 
-type errResponse struct {
-	Err string `json:"error,omitempty"`
-}
-
-func errorHandler(ctx context.Context, _ *runtime.ServeMux, marshaler runtime.Marshaler, writer http.ResponseWriter, _ *http.Request, err error) {
```

Comment:
What is the reason the errorHandler is removed?

Reply:
Same as above, In grpc-gateway v2, Error handling configuration has been overhauled.
_runtime.HTTPError, runtime.OtherErrorHandler, runtime.GlobalHTTPErrorHandler, runtime.WithProtoErrorHandler are all gone. Error handling is rewritten around the use of gRPCs Status types._
Custom Error handling function may not be required anymore.